src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/EnvelopeEventProcessorTest.java

```diff
@@ -123,24 +134,43 @@ public void should_dead_letter_the_message_when_unrecoverable_failure() throws E
 
     @Test
     public void should_not_finalize_the_message_when_recoverable_failure() throws Exception {
-        // given
-        reset(eventPublisherContainer);
-        willThrow(new RuntimeException("test exception"))
-            .given(eventPublisherContainer)
-            .getPublisher(any(), any());
+        verifyNoInteractionsWhenMessageProcessingFails(messageOperations);
+    }
 
+    @Test
+    public void should_not_do_anything_in_notify_exception() {
+        // when
+        processor.notifyException(null, null);
```

Comment:
this does not prove 'should not do anything'

Reply:
I agree the name of the method is wrong. Didn't touch it - it's just how the diff is presented.

Reply:
Oh, actually, I added `exception` at the end

Reply:
Reverted - I don't want to mix changes in this already not-so-small PR. Now I haven't touched it. But yes, I agree this test should either go away or say `no exception is thrown...`.